src/app/pages/deprecation/deprecation.css

```diff
@@ -0,0 +1,17 @@
+.released {

```

Comment:
interesting to put css file in your features folder.

but how is this rendered?
-> only content from the web folder can be served.
-> I would simple inline this css in your page. makes it a lot easier.